src/App.jsx

```diff
@@ -26,6 +30,11 @@ const App = () => (
       <Route path="/article/:slug" exact component={withAlert(ReadUpdateArticleContainer)} />
       <Route path="/signup/success" exact component={SignUpSuccess} />
       <Route path="/verify/:token" exact component={VerificationContainer} />
+      <Route path="/reset_password/:token" exact component={ResetPasswordContainer} />
+      <Route path="/profiles/:username" exact component={Profile} />
+      <Route path="/update-info/:username" exact component={UpdateProfile} />
```

Comment:
Your routes should be consistent. 
Since you are dealing with profiles, I expect this route to be something like: 
```
/profiles/update/:username
```

Read up on best practices of route naming here:
https://restfulapi.net/resource-naming/

Reply:
Ok cool, on it.